<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>T1WrenchMainWindow</class>
 <widget class="QMainWindow" name="T1WrenchMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>552</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>T1WrenchMainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>771</width>
      <height>441</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>聊天专区</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>521</width>
        <height>391</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPlainTextEdit" name="phoneTextEdit"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="qqButton">
           <property name="text">
            <string>微信、QQ</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cellMailButton">
           <property name="text">
            <string>回邮件</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="t1SmsButton">
           <property name="text">
            <string>回短信</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="weiboButton">
           <property name="text">
            <string>微博</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="googlePlusButton">
           <property name="text">
            <string>Google+</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="toPhoneClipboardButton">
           <property name="text">
            <string>-&gt;剪贴板</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="fromPhoneClipboardButton">
           <property name="text">
            <string>&lt;-剪贴板</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="qqHintLabel">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>221</width>
        <height>391</height>
       </rect>
      </property>
      <property name="text">
       <string>适用场景示意图</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>751</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="cmdOutputEdit">
       <property name="minimumSize">
        <size>
         <width>580</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>580</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="adbStateLabel">
       <property name="text">
        <string>Disconnected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="adbStateIndicator">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="T1Wrench.qrc">
         <normaloff>:/images/red.png</normaloff>
         <disabledoff>images/red.png</disabledoff>
         <disabledon>images/green.png</disabledon>:/images/red.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="T1Wrench.qrc"/>
 </resources>
 <connections/>
</ui>
